src/App.tsx

```diff
@@ -4,7 +4,9 @@ import { useHistory } from 'react-router-dom';
 import { Routes } from './Routes';
 import './App.scss';
 
-const App = (props) => {
+declare var insights: any;
```

Comment:
Out of curiosity, do you know if the upstream `insights` library coming in here is written in TS?

Reply:
That I'm not too sure about. Let me dig into that a bit more

- [ ] Investigate if the upstream insights library coming in here is written in TS